src/content/web/get-started.md

```diff
@@ -43,9 +43,15 @@ $ dart create -t web quickstart
 
 <i class="material-symbols">web</i>
 To create the same web app from an IDE that has Dart integration,
-create a project using the template named **Bare-bones Web App**.
+[create a project]({{site.flutter}}/tools/vs-code#creating-a-new-project)
+using the template named **Bare-bones Web App**.
+
+The web app template imports [`package:web`][], Dart's powerful and concise web
+interop solution built for the modern web. You can learn more about it
+[here](/interop/js-interop/package-web).
```

Comment:
Try to avoid "here" if possible:

```suggestion
interop solution built for the modern web. To learn more about it, check out the
[web interop overview](/interop/js-interop/package-web).
```

Reply:
Thank you!